runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -282,6 +283,16 @@ else if (client.getWidget(WidgetInfo.QUEST_COMPLETED_NAME_TEXT) != null)
 		}
 	}
 
+
+	@Subscribe
+	public void onLocalPlayerDeath(LocalPlayerDeath death)
+	{
+		if (config.screenshotPlayerDeads())
+		{
+			takeScreenshot("Died on " + LocalDate.now());
```

Comment:
You should probably follow other screenshots conventions, e.g this should be "Death " + format(new Date()) i think